metricbeat/module/system/entropy/_meta/fields.yml

```diff
@@ -0,0 +1,19 @@
+- name: entropy
+  type: group
+  description: >
+    entropy
+  fields:
+    - name: entropy
+      type: group
+      description: >
+        Available system entropy
+      fields:
+        - name: available_bits
+          type: long
+          description: >
+            The available bits of entropy
+        - name: pct
+          type: scaled_float
+          type: percentage
```

Comment:
Should this be `format: percent` instead? I've never used `type: percentage` before so this is just a question 😄

Reply:
Ooops

So that should be `format` but it looks like we have `percent` and `percentage` in the source.

Reply:
hmmm by looking at https://github.com/elastic/beats/blob/master/metricbeat/module/system/core/_meta/fields.yml#L15, it's `format: percent` hmm